code/lib/docs-tools/src/argTypes/jsdocParser.ts

```diff
@@ -164,6 +178,16 @@ function extractParam(tag: doctrine.Tag): ExtractedJsDocParam {
   return null;
 }
 
+function extractDeprecated(tag: doctrine.Tag): ExtractedJsDocDeprecated {
+  if (tag.title != null) {
+    return {
+      name: tag.description,
```

Comment:
I would think this is a "description" rather than a "name"? That's at least how we use it in our TS prop types, and what I can gather from JSDoc specs:

```ts
  /**
   * @deprecated Manually specifying description type is deprecated. See https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#description-block-parametersnotes-and-parametersinfo
   */
  type?: DescriptionType;
```